styles/styles.css

```diff
@@ -0,0 +1,348 @@
+body {
+  margin: 0;
+  width: 100%;
+}
+
+/*
+header {
+  background-color: #0A3C61;
+  height: 21rem;
+  text-align: left;
+  display: flex;
+  justify-content: center;
+}
+*/
+header {
+  background-color: #0A3C61;
+  height: 24rem;
+  justify-content: flex-start;
+
+}
+
+header h1 {
+  color: black;
+  position: absolute;
+  left: 13rem;
+  top: 1rem;
+  font-size: 60px;
+  font-family: sans-serif;
+}
+
+header h2 {
+  color: white;
+  position: absolute;
+  font-size: 25px;

```

Comment:
Make sure to use rem's for font sizes. You can just do math 25px / 16 = 1.5625rem (you could round down to 1.5 rem if you'd like.

Reply:
replaces with rem